src/am_map.c

```diff
@@ -2309,6 +2317,25 @@ void AM_Drawer (void)
     AM_clearFB(mapcolor_back);       //jff 1/5/98 background default color
     pspr_interp = false;
   }
+  // [Alaux] Dark automap overlay
```

Comment:
This is the exact same code as used in `m_menu.c:M_Drawer()`. Could you please factor it out into a common helper function, most approriately in `v_video.c`?

Reply:
Also, what if shaded automap overlay is enabled and then the menu with shaded background is opened? Would the screen get shaded twice then?

Reply:
Indeed, the shading effect would be applied twice. I was aware of this but didn't think much about it. Should be able to change it as I make the helper function.